handlers/Base/index.js

```diff
@@ -24,25 +25,7 @@ class AppBase extends Component {
   }
 
   render(): ReactElement {
-    var tags = [
-      {name: 'title', content: 'Skookum'},
-      {name: 'description', content: 'Skookum is a full service software development company with capabilities spanning product strategy, UI/UX design, development and support.'},
-      {itemProp: 'name', content: 'Skookum Digital Works'},
-      {itemProp: 'description', content: 'Skookum is a full service software development company with capabilities spanning product strategy, UI/UX design, development and support.'},
-      // {itemProp: 'image', content: skookumLogoUrl},
-      {name: 'twitter:card', content: 'summary'},
-      {name: 'twitter:site', content: '@skookum'},
-      {name: 'twitter:title', content: 'Skookum'},
-      {name: 'twitter:description', content: 'Skookum is a full service software development company with capabilities spanning product strategy, UI/UX design, development and support.'},
-      {name: 'twitter:creator', content: '@skookum'},
-      {name: 'twitter:image', content: 'http://pbs.twimg.com/profile_images/648570516459008000/26E19ffH.png'},
-      {property: 'og:title', content: 'Skookum'},
-      {property: 'og:type', content: 'website'},
-      {property: 'og:url', content: 'http://skookum.com/'},
-      {property: 'og:image', content: 'http://pbs.twimg.com/profile_images/648570516459008000/26E19ffH.png'},
-      {property: 'og:description', content: 'Skookum is a full service software development company with capabilities spanning product strategy, UI/UX design, development and support.'},
-      {property: 'og:site_name', content: 'Skookum Digital Works'},
-    ];
+    var tags = getDefaultTags();
```

Comment:
It's now setup so that these are all the default values and the title and description can be updated dynamically via contentful for every page. Blogs update most of them separately. But just realized that I think I've only set it up to update the standard title and description, and not the twitter, og, or structured data ones. Checking with Rob now on that part.